src/webapp/azext_webapp/custom.py

```diff
@@ -193,6 +193,15 @@ def create_deploy_webapp(cmd, name, location=None, dryrun=False):
     logger.warning("All done.")
     return create_json
 
+def _ping_scm_site(cmd, resource_group, name):
```

Comment:
can you update the create_deploy_webapp to use the helper directly as well.

Reply:
will do